tests/scripts/all.sh

```diff
@@ -4642,6 +4645,9 @@ component_check_test_helpers () {
     python3 -m unittest tests/scripts/translate_ciphers.py 2>&1
 }
 
+# Include tests for data files
+source tests/scripts/regenerate_data_files_tests.sh
```

Comment:
Because of the position of this command (it comes before 'pre_check_environment'), when running the script from an incorrect path, instead of getting the "Must be run from mbed TLS root" error, I get "tests/scripts/regenerate_data_files_tests.sh: No such file or directory" as an error. Please fix this.

Reply:
Got it

Reply:
Now I don't get any error whatsoever